packages/perseus-core/src/widgets/numeric-input/numeric-input-util.ts

```diff
@@ -0,0 +1,26 @@
+import type {PerseusNumericInputWidgetOptions} from "@khanacademy/perseus-core";
+
+/**
+ * For details on the individual options, see the
+ * PerseusNumericInputWidgetOptions type
+ */
+type NumericInputPublicWidgetOptions = {
+    labelText?: PerseusNumericInputWidgetOptions["labelText"];
+    size: PerseusNumericInputWidgetOptions["size"];
+    coefficient: PerseusNumericInputWidgetOptions["coefficient"];
+    rightAlign?: PerseusNumericInputWidgetOptions["rightAlign"];
+    static: PerseusNumericInputWidgetOptions["static"];
```

Comment:
I left out both `answers` and `answerForms` as they both appear to contain information related to the correct answer. 'answerForms` even has a note that it's supposedly used in examples, but might not even be used anymore.

Reply:
I think answerForms should actually be considered public information. See e.g. the `examples()` method on `NumericInput`, which apparently generates learner-facing instructions based on the `answerForms` that are accepted.

Reply:
Related to this, I added @SonicScrewdriver as a reviewer (and pinged her) since she's just been looking at this.

Reply:
My concern was that answerForms has information that could tell the user what to do. Specifically, the answerFrom names field: 

```
    answerForms: ReadonlyArray<{
        simplify: "required" | "correct" | "enforced" | null | undefined;
        name: "integer" | "decimal" | "proper" | "improper" | "mixed" | "pi";
    }>;
```

Maybe that information is supposed to be available to the user?

Reply:
Yes, the answer forms are displayed to the learner. Example:

<img width="275" alt="Screen Shot 2025-01-31 at 8 43 59 AM" src="https://github.com/user-attachments/assets/4cb68100-4b3c-462a-83c1-5091f1a839bf" />

Reply:
Yes! The answer forms are displayed to the learner, and should already be specified to the user through how the question is written. 

> For example: If the question wants a fraction answer, the user will be prompted about this in a secondary fashion as well, either in the title of their course ("Improper Fractions") or directly in the question text. ("Please write your answer as an improper fraction"). 

The tool tips are just helpful reminders for users, particularly young ones. I don't think this could be something that anyone could use to cheat or get ahead on a question, as the user should never be surprised about which answer format the question requires. 

As a result, I think it's safe to keep the answerForms public!

Reply:
Awesome! Thank you both! Just added them back in :D